grpc/src/compiler/go_generator.cc

```diff
@@ -84,7 +84,7 @@ void GenerateImports(grpc_generator::File *file, grpc_generator::Printer *printe
 	}
 	printer->Print("import (\n");
 	printer->Indent();
-	printer->Print(vars, "$context$ \"golang.org/x/net/context\"\n");
```

Comment:
this is a file we copy from gRPC, sp ideally it be fixed upstream as well..

Reply:
@felix any response to this?

Reply:
It depends on how long we have to wait. Can't this be merged here so FB's tooling keeps pace and then it will be moot once upstream changes?